configs/data-driven-forms.json

```
@@ -17,5 +17,7 @@
   "conversation_id": [
     "952910917"
   ],
-  "nb_hits": 3573
+  "nb_hits": 3573,
+  "js_render": true,
+  "js_wait": 2
```

Would it be possible to reduce as much as possible the time to wait?

---

I would like to, is there any way how to determine it? Or should I try some smaller value?

---

The query params should be available almost instantly. The issue is that the router needs to just access the window property and set query values. It should take less than a second IMO.

---

```suggestion
  "js_wait": 1
```
Let's give it a try with 1.